src/System.Private.Uri/src/System/Uri.cs

```diff
@@ -2109,12 +2109,18 @@ private unsafe ParsingError PrivateParseMinimal()
                             _flags |= Flags.UncPath;
                             idx = i;
                         }
+                        else if (!IsWindowsSystem && _syntax.InFact(UriSyntaxFlags.FileLikeUri) && i - idx == 3 && pUriString[i - 1] == '/')
+                        {
```

Comment:
@tmds We should fix all cases where after detecting that it's a valid file Unix Uri, the path backslashes are not converted to forward slash. The behavior of backward to forward slash with and without authority is confusing to be different. We should get all cases to behave the same, not just file:///. Is there something special with file:/// that this behavior should be inapplicable for other cases?